### PR TITLE
chore: Updating list of documents

### PR DESCRIPTION
**Pull request by m-strzelczyk**
Adding a new link to README pointing to documentation that uses samples from this folder.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: compute/metadata/README.md [hunk 1]
```diff
@@ -11,5 +11,6 @@ These samples demonstrate interacting with the Compute Engine metadata service.
 These samples are used on the following documentation page:
 
 > https://cloud.google.com/compute/docs/metadata
```
Comment (dandhlee) [inline compute/metadata/README.md:13]: Could you update this link? It redirects to `https://cloud.google.com/compute/docs/storing-retrieving-metadata` which I think it still redirects to the link you want but let's update the original link as well.

File: compute/metadata/README.md [hunk 2]
```diff
@@ -11,5 +11,6 @@ These samples demonstrate interacting with the Compute Engine metadata service.
...
+> https://cloud.google.com/compute/docs/instances/verifying-instance-identity
```
Comment (dandhlee) [inline compute/metadata/README.md:14]: The two links show up side by side, could you fix the formatting? Reference: https://cloud.google.com/compute/docs/storing-retrieving-metadata

**Review by dandhlee — CHANGES_REQUESTED** [review]
Thanks for the update! Please see comments below :)

**Review by dandhlee — APPROVED** [review]
LGTM.

**Review by engelke — APPROVED** [review]

**Pull request merged**